scripts/buildRpm.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+
+PACKAGE=CodeCoverage
+VERSION=1.0
+
+PWD=`pwd`
+CWD=$PWD/$PACKAGE
+DISTDIR=$CWD/dist/$PACKAGE
+PATH=$PATH:/usr/local/probe/bin:$PATH
```

Comment:
This needs to be cleaned up. PATH probably doesn't need to be set here and I don't think DISTDIR is needed.